runtime/queries/matlab/injections.scm

```diff
@@ -0,0 +1 @@
+(comment) @comment
```

Comment:
```suggestion
((comment) @injection.content
 (#set! injection.language "comment"))
```

We follow tree-sitter's injection format: https://tree-sitter.github.io/tree-sitter/syntax-highlighting#language-injection. I believe this is neovim's style. (I'm not sure if emacs follows nvim here.)